data/json/items/tools.json

```diff
@@ -8777,5 +8777,45 @@
       }
     ],
     "flags": [ "LIGHT_3", "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "pallet_full",
```

Comment:
I'd recommend making these item IDs more specific to Adobe, as pallets could be used for many things

Reply:
I removed the pallet recipe and made the bricks just dry by themselves. Sorry, was away for a few hours and missed these posts because I haven't slept. Yes, I'll look into adding different stats... does looks_like function by making a 'brick wall' terrain that has different underlying stats to the regular one?

Reply:
From my investigation into the topic, adobe has great compressive strength, much like rammed earth, but doesn't stand up well to the damp apparently without a layer of stucco. It's not the issue of it being stronger, really, so much as I need to add sand/lime to the construction so that the outside is (abstractly) sealed against the New England damp by a coating of plaster.